cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -33,8 +33,7 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         query("CREATE USER jake SET PASSWORD 'abc' CHANGE REQUIRED SET STATUS SUSPENDED", ResultAssertions((r) => {
           assertStats(r, systemUpdates = 1)
         })) {
-          p("Nothing is returned from this query, except the count of system database changes made.")
-          resultTable()
+          statsOnlyResultTable()
```

Comment:
I still think there are empty `{}` which would benefit from using the `statsOnlyResultTable()` as well but this is a good start :)